### PR TITLE
Rename functions and parameters dealing with AccessControlLevel

### PR DESCRIPTION
**Pull request by webwarrior-ws**
Renamed some functions and/or parameters
related to AccessControlLevel to make their
intent more clear.

### CODE REVIEW

File: src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs [hunk 1]
```diff
@@ -209,15 +209,15 @@ let getAccessibility (syntaxArray:AbstractSyntaxArray.Node []) i =
             | ExceptionRepresentation(SynExceptionDefnRepr.SynExceptionDefnRepr(_, _, _, _, access, _))
             | Pattern(SynPat.Named(_, _, _, access, _))
             | Pattern(SynPat.LongIdent(_, _, _, _, access, _)) ->
-                getAccessibility (isSynAccessPublic access) isPrivateWhenReachedBinding node.ParentIndex
+                getAccessibility (resolveAccessControlLevel access) isPrivateWhenReachedBinding node.ParentIndex
```
Comment (knocte) [inline src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs:212]: @webwarrior-ws does this build actually? or is there another function named getAccessibility that needs to be renamed?

File: src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs [hunk 2]
```diff
@@ -189,8 +189,8 @@ type AccessControlLevel =
     | Private
     | Internal
 
-let getAccessibility (syntaxArray:AbstractSyntaxArray.Node []) i =
-    let isSynAccessPublic = function
+let getAccessControlLevel (syntaxArray:AbstractSyntaxArray.Node []) i =
+    let resolveAccessControlLevel = function
```
[the remark this answers is not in the document]
Comment (webwarrior-ws) [reply src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs:193]: No, it's a function, so it should be a verb

[the remark this answers is not in the document]
Comment (knocte) [reply src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs:193]: ah good point

File: src/FSharpLint.Core/Rules/Conventions/Naming/ParameterNames.fs [hunk 3]
```diff
@@ -40,7 +40,7 @@ let private getIdentifiers (args:AstNodeRuleParams) =
         if not (isLiteral attributes) then
             match identifierTypeFromValData valData with
             | Value | Function ->
-                let accessibility = getAccessibility args.SyntaxArray args.NodeIndex
+                let accessibility = getAccessControlLevel args.SyntaxArray args.NodeIndex
```
[the remark this answers is not in the document]
Comment (webwarrior-ws) [reply src/FSharpLint.Core/Rules/Conventions/Naming/ParameterNames.fs]: Renamed to `accessControlLevel`

File: src/FSharpLint.Core/Rules/Conventions/Naming/InternalValuesNames.fs [hunk 4]
```diff
@@ -6,7 +6,7 @@ open FSharpLint.Framework.AstInfo
 open FSharpLint.Framework.Rules
 open FSharpLint.Rules.Helper.Naming
 
-let private getValueOrFunctionIdents typeChecker isPublic pattern =
+let private getValueOrFunctionIdents typeChecker accessibility pattern =
```
Comment (knocte) [inline src/FSharpLint.Core/Rules/Conventions/Naming/InternalValuesNames.fs:9]: shouldn't this be called accessControlLevel instead?

**Pull request merged**